### PR TITLE
Feature/start controller

### PR DESCRIPTION
**Pull request by JoeChapman**
Add and test start-controller
eslint was not running or passing because elsint homeoffice config was not saved as a dep. Added.

### CODE REVIEW

File: lib/start-controller.js [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+
+module.exports = class StartController extends BaseController {
+
+  constructor(options) {
+    super(options);
+  }
+
+  getValues(req) {
+    req.sessionModel.reset();
+    super.successHandler.apply(this, arguments);
```
Comment (joefitter) [inline lib/start-controller.js]: when using super.methodName, you don't need to pass the scope. I know this way you can just pass `arguments` but I think I prefer the explicit 

``` js
super.successHandler(req, res, callback);
```

with the extra args in the class method, thoughts?

Comment (JoeChapman) [inline lib/start-controller.js]: Yeah, sorry I hadn't even noticed. Frankly this was a bit of a copy and paste and I wans't paying quite enough attention.

**Comment by joefitter** [comment]
@JoeChapman LGTM but shouldn't the ESLint stuff be in a separate PR?

**Comment by JoeChapman** [comment]
> shouldn't the ESLint stuff be in a separate PR?

Perhaps, but the start-controller could not be committed without the eslint fix.

**Comment by joefitter** [comment]
Cool @easternbloc @daniel-ac-martin another 👍 ?

**Comment by easternbloc** [comment]
I'm happy enough as it's as separate commits. LGTM

**Comment by easternbloc** [comment]
Ah crap. I just realised. DOCS?

**Comment by JoeChapman** [comment]
doing^

**Pull request merged**